src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunPlugin.java

```diff
@@ -12,7 +12,6 @@
 
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
-
```

Comment:
We don't want to mix imports with different purposes.

Reply:
it was like that originally